src/pages/Home/Home.jsx

```diff
@@ -11,6 +12,9 @@ const Home = () => {
   }
   return (
     <Layout isWithMenu={true}>
+      <Button buttonName="map-navigation button--plus" buttonText="+" />
+      <Button buttonName="map-navigation button--minus" buttonText="-" />
```

Comment:
Are these plus/minus styles being applied? I think it needs to passed through as `button-minus` and `button-plus`. the Button component adds the `--` itself